Sources/Segment/Timeline.swift

```diff
@@ -210,18 +210,28 @@ extension DestinationPlugin {
         }
         return result
     }
+    
+    internal func isDestinationEnabled(integrations: JSON?) -> Bool {
+        var customerDisabled = false
+        if let disabled: Bool = integrations?.value(forKeyPath: KeyPath(self.key)), disabled == false {
+            customerDisabled = true
+        }
+        
+        var hasSettings = false
+        if let settings = analytics?.settings() {
+            hasSettings = settings.hasIntegrationSettings(forPlugin: self)
+        }
+        
+        return (hasSettings && customerDisabled == false)
```

Comment:
I'd prefer this line as `hasSettings == true && customerDisabled == false` OR `hasSettings && !customerDisabled` so we aren't mixing semantics.